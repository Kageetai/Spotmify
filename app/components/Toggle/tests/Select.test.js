import React from 'react';
import renderer from 'react-test-renderer';

import Select from '../Select';

describe('<Select />', () => {
  it('should match the snapshot', () => {
    const wrapper = renderer.create(<Select />).toJSON();
    expect(wrapper).toMatchSnapshot();
  });
});
