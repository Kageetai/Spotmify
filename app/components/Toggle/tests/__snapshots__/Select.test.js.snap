// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Select /> should match the snapshot 1`] = `
.c0 {
  line-height: 1em;
  height: 20px;
}

<select
  className="c0"
/>
`;
